### PR TITLE
Update dark theme docs

### PR DESCRIPTION
**Pull request by stamminator**
- Remove deprecated sys.dark reference from docs
- List current theme.dark options

### CODE REVIEW

**Comment by Xhib-ba** [comment]
> Remove deprecated `sys.dark` reference from docs

`sys.dark` per se works fine. It just does not sit well with `dark=...`  in the `theme{}` section. Consider this for example:
```js
item (
       title=if(sys.dark,"Dark Mode \tEnabled","Dark Mode \tDisabled")
       icon=\uE1C9
      )
```
Screenshot 2024-07-10 210846
Screenshot 2024-07-10 210900

File: docs/functions/sys.html [hunk 1]
```diff
@@ -22,13 +22,6 @@ <h5>sys.is64</h5>
 	<code>sys.is64</code>
 </section>
 
-<section id="sys.dark" class="my-5">
-	<h5>sys.dark</h5>
-	<p>Check if dark mode is enabled.</p>
-	<p>Syntax</p>
-	<code>sys.dark</code>
-</section>
-
```
Comment (Xhib-ba) [inline docs/functions/sys.html:31]: Can be retained. See https://github.com/moudey/Shell/pull/481#issuecomment-2220988856

**Comment by moudey** [comment]
`sys.dark` not deprecated!

`sys.dark` Returns a boolean value for the system's dark mode
`theme.dark` Returns a boolean value for the Shell's dark mode

Please try the latest [Canary](https://nilesoft.org/download#canary) version 1.16

**Comment by stamminator** [comment]
Thanks for the correction, guys! I just reverted the `sys.dark` change. I will test out version 1.16 tonight.

**Comment by stamminator** [comment]
> `sys.dark` not deprecated!
> 
> `sys.dark` Returns a boolean value for the system's dark mode `theme.dark` Returns a boolean value for the Shell's dark mode
> 
> Please try the latest [Canary](https://nilesoft.org/download#canary) version 1.16

`sys.dark` works for me in the canary build

**Pull request merged**